pkg/agent/openflow/pipeline.go

```diff
@@ -875,33 +876,68 @@ func (c *client) arpNormalFlow(category cookie.Category) binding.Flow {
 
 // conjunctionActionFlow generates the flow to jump to a specific table if policyRuleConjunction ID is matched. Priority of
 // conjunctionActionFlow is created at priorityLow for k8s network policies, and *priority assigned by PriorityAssigner for CNP.
-func (c *client) conjunctionActionFlow(conjunctionID uint32, tableID binding.TableIDType, nextTable binding.TableIDType, priority *uint16) binding.Flow {
+func (c *client) conjunctionActionFlow(conjunctionID uint32, tableID binding.TableIDType, nextTable binding.TableIDType, priority *uint16, enableLogging bool) binding.Flow {
 	var ofPriority uint16
 	if priority == nil {
 		ofPriority = priorityLow
 	} else {
 		ofPriority = *priority
 	}
 	conjReg := IngressReg
-	if tableID == EgressRuleTable {
+	if tableID == EgressRuleTable   {
 		conjReg = EgressReg
 	}
+	for _, table := range GetCNPEgressTables() {
+		if tableID == table {
+			conjReg = EgressReg
+		}
+	}
+	if enableLogging {
+		return c.pipeline[tableID].BuildFlow(ofPriority).MatchProtocol(binding.ProtocolIP).
+			MatchConjID(conjunctionID).
+			MatchPriority(ofPriority).
+			Action().LoadRegRange(int(conjReg), conjunctionID, binding.Range{0, 31}). // Traceflow.
+			Action().LoadRegRange(int(DispositionReg), 1, binding.Range{0, 31}). //CNP
```

Comment:
define the actions as dispositionAllow = 1,  dispositionDrop = 2 for example as const in this pkg, and we can refer to those in packetin.go as openflow.dispositionAllow